docs/UserGuide.md

```diff
@@ -14,29 +13,13 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `EzSchedule.jar` from [here](https://github.com/AY2223S2-CS2103-W17-1/tp).
```

Comment:
Can add _/release_ to the back of the link so that it looks like https://github.com/AY2223S2-CS2103-W17-1/tp/releases (points directly to the release page)